### ISSUE TITLE
Upgrade dependencies

### ISSUE THREAD

**Issue by matheus23** [fission-suite/landing-page#21]
I noticed some dependencies are outdated. I'd be happy to do an upgrade and am fairly sure, it'd go smoothly.

```
$ npm outdated
Package                      Current  Wanted  Latest  Location
@fullhuman/postcss-purgecss    2.1.2   2.3.0   2.3.0  global
autoprefixer                   9.7.6   9.8.6   9.8.6  global
elm-git-install                0.1.2   0.1.3   0.1.3  global
elm-pages                      1.3.0   1.3.0   1.4.2  global
fission-kit                    0.2.0     git     git  global
postcss                       7.0.27  7.0.32  7.0.32  global
postcss-elm-tailwind           0.8.0   0.8.0  0.10.0  global
sharp                         0.24.1  0.24.1  0.25.4  global
tailwindcss                    1.2.0   1.6.2   1.6.2  global
```

```
$ elm-json upgrade --unsafe

-- PACKAGE UPGRADES FOUND ------------------------------------------------------

I want to make some changes to your direct dependencies

- [CHG] dillonkearns/elm-pages 4.0.1 -> 6.0.0
- [CHG] elm/core 1.0.4 -> 1.0.5
- [CHG] elm-community/html-extra 3.3.0 -> 3.4.0
- [CHG] elm-community/list-extra 8.2.3 -> 8.2.4
- [CHG] elm-community/result-extra 2.3.0 -> 2.4.0
- [CHG] feathericons/elm-feather 1.4.0 -> 1.5.0
- [CHG] justinmimbs/date 3.2.0 -> 3.2.1
- [CHG] turboMaCk/any-dict 2.1.0 -> 2.3.0
```

Still wanted to check in with you, if such a change would be desired. At the moment I don't think I'll need any new features in the upgraded elm or packages or tailwindcss.

**Comment by icidasset** [comment]
Yeah, sure, should be fine 👍 

If you're up for it, I did want to switch out the PostCSS/Tailwind setup with https://github.com/icidasset/elm-tailwind-css which we use in other projects as well (or at least starting to). I kept doing the same thing over and over again for each project, so I made a thing.

**Comment by matheus23** [comment]
Hmm interesting. Have you seen [postcss-elm-css-tailwind](https://discourse.elm-lang.org/t/generate-your-tailwind-utilities-as-elm-css-functions/6076)?

It doesn't generate a Tailwind.elm file quite as big. It's only ~18k LOC. And instead of generating lots of variants for selectors like `sm:` or `hover:` you simply use elm-css' capabilities for that. Seemed quite nice to me. Might be a little experimental though.

**Comment by icidasset** [comment]
Yeah I saw that, great stuff 👍 
That said, it's indeed a bit too experimental at this stage.
We can explore this later though.

**Comment by matheus23** [comment]
Addressed this in #23

**Comment by matheus23** [comment]
Closed by 41df11dbe125736edcfa33d0ef48e1c10d0de299
